### ISSUE TITLE
No gmsh:physical and gmsh:geometrical in cell_data

### ISSUE THREAD

**Issue by mingdeyu008** [nschloe/pygmsh#223]
I am new to _pygmsh_ and would like to reproduce the result on this website:

[https://github.com/geodynamics/specfem3d/tree/master/EXAMPLES/Gmsh_simple_lddrk](url)

However, I found that using the current versions of _pygmsh_ and _meshio_, there are no _gmsh:physical_ and _gmsh:geometrical_ in the _cell_data_ generated from the _pygmsh.generate.mesh_. 

I wonder how these two information can be extracted into _cell_data_ using the current versions of _pygmsh_ and _meshio_?

**Comment by mingdeyu008** [comment]
It seems the problem of the Gmsh version. Installing the older version of Gmsh rectifies the problem.

**Comment by mingdeyu008** [comment]
I removed the `num_quad_lloyd_steps` from the arguments of `pygmsh_generate_mesh` and also changed the outputs of `meshio.read` (see line 44 in `Gmsh2specfem.py`) from the five separate variables (`points, cells, point_data, cell_data, field_data`) to one single variable and then extract the five elements from it. I also changed the `file_format` to `gmsh4-ascii` because it seems the current version of meshio does not support `gmsh-ascii`. After these changes, implementing `mesh_example_with_gmsh.py` will run well and direct to `Gmsh2specfem.py` until it raises an error reporting that `gmsh:physical` cannot be found in the `cell_data`. But this error is solved by installing the older version (version 3.0) of Gmsh. I compared the results to the reference files in "specfem3d/EXAMPLES/Gmsh_simple_lddrk/" and can see they are same. So I think the issue is whether `gmsh:physical` is also not generated in `cell_data` for the current version of Gmsh...
 
BTW, I change `meshio.write` to  `meshio.write_points_cells` in all places because in current version of meshio `meshio.write` takes a single mesh argument rather than the five separate elements.
